config/config.schema.yaml

```diff
@@ -207,3 +207,27 @@ properties:
         type: array
         items:
           type: string
+
+  # Auth Server Configuration (optional)
```

Comment:
> This part of the config is only used by the Envoy Auth Server, not in PATH itself.

Given that we have 4 config file:
1. Is there not a better place for this:
2. Separating this into its own config file?

We don't have to do it in this PR, but I'd like to track it in a ticket or TODO if (for example) a separate config file is more appropriate.